README.md

```diff
@@ -45,7 +45,7 @@ A [breakdown of sections](SPRINT-STRUCTURE.md) according to the structure of an
 * [CI](continuous-integration/readme.md)
 * [CD](continuous-deployment/readme.md)
 * [Code Reviews](code-reviews/readme.md)
-* [Observability](observability/readme.md)
+* [Observability](observability/README.md)
```

Comment:
We have all readme's lower case so I would keep it consistent if possible - ie. Rename observability/README.md